GraphQL-TSC.md

```diff
@@ -210,6 +217,20 @@ TSC members are not required to vote. There are three ways an _attending_ member
 
 Note: Non-attending members do not count towards quorum, and should they not vote are effectively recused.
 
+### All possible vote types
+
+And their impact on what counts towards each factor of the voting process.
+
+| Vote                            | Counts as _attending_ | Counts towards _quorum_ | Counts towards a choice |
+| ------------------------------- | :-------------------: | :---------------------: | :---------------------: |
+| Attending member choice         |          ✅           |           ✅            |           ✅            |
+| Attending member present        |          ✅           |           ✅            |           ❌            |
+| Attending member absent         |          ✅           |           ❌            |           ❌            |
+| Attending member abstain        |          ✅           |           ❌            |           ❌            |
+| Attending member recused        |          ❌           |           ❌            |           ❌            |
+| "Non-attending member" choice   |          ❌           |           ❌            |           ✅            |
+| "Non-attending member" non-vote |          ❌           |           ❌            |           ❌            |
```

Comment:
It took me a moment to digest this, so here's a brief summary of my understanding:

"Attending member" and "non-attending member" refer to whether the TSC member has attended recent WG meetings or not (doesn't relate to attendance at the vote).

For the vote you can have one of 4 statuses (the default being "absent"):

- choice - you say "yes" / "no" (or whatever is appropriate for the vote) and your vote is counted
- present - you say "present" and thus you count for reaching quorum but not as a vote
  - helps the vote progress
- absent/abstain (equivalent) - you don't count towards quorum nor towards the vote
  - If you don't want to vote for/against something but you want the vote to conclude then you should say "present" instead so you count towards quorum
- recused - if you have a conflict of interest then you should recuse yourself and we all pretend you don't and never have existed for the purposes of the vote/quorum/etc 😉 

More about present/absent/abstain/recused here: https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#non-votes